ngraph/core/src/op/abs.cpp

```diff
@@ -36,8 +36,12 @@ op::Abs::Abs(const Output<Node>& arg)
 
 shared_ptr<Node> op::Abs::clone_with_new_inputs(const OutputVector& new_args) const
 {
-    check_new_args_count(this, new_args);
-    return make_shared<Abs>(new_args.at(0));
+    NGRAPH_OP_SCOPE(v0_Abs_clone_with_new_inputs)
+    {
+        check_new_args_count(this, new_args);
+        return make_shared<Abs>(new_args.at(0));
+    }
+    throw ngraph_error("v0_Abs_clone_with_new_inputs is disabled!");
```

Comment:
does not throwing exceptions increase binary size in a default build?

Reply:
I checked the library size on macOS and didn't see any increases.

In general I can change the OV_SCOPE macro in order to throw `std:: runtime_error` in case if the scope was disabled. Is it ok for you?

Reply:
yes, see the proposal below. it's about the same

Reply:
Fixed